core/state/trie_prefetcher_extra_test.go

```diff
@@ -126,7 +126,7 @@ func BenchmarkPrefetcherDatabase(b *testing.B) {
 		if previous != root {
 			require.NoError(db.TrieDB().Dereference(previous))
 		} else {
-			b.Fatal("root did not change")
+			b.Fail()
```

Comment:
```suggestion
require.Equal(root, previous)
require.NoError(db.TrieDB().Dereference(previous))
```
Sorry the indentation is wrong